lib/grafana_matrix/server.rb

```diff
@@ -77,10 +77,10 @@ def authorized(user, pass)
         next unless client.is_a? MatrixSdk::Api
 
         client.send_message_event(room, 'm.room.message',
-                                  msgtype: rule.msgtype,
+                                  {msgtype: rule.msgtype,
                                   body: plain,
                                   formatted_body: html,
-                                  format: 'org.matrix.custom.html')
+                                  format: 'org.matrix.custom.html'})
```

Comment:
Could you add whitespace inside the braces, and also indent the keys two spaces after that, so that they're aligned.
Going to be cleaning this up and enabling CI with rubocop - and that's going to be an offense otherwise.